docs/src/pages/[platform]/components/alert/react.mdx

```diff
@@ -22,11 +22,11 @@ import {
 Import the Alert component and styles.
 
 <Example>
-  <View>
+  <ExamplePreview>
     <DefaultAlertExample />
-  </View>
+  </ExamplePreview>
   
-  <ExampleCode>
+  <ExampleCode reactExport='DefaultAlertExample'>
```

Comment:
We need the export name in order to generate code that imports it. See `openInStackblitz`